apps/devchoices-next/pages/question/[slug].tsx

```diff
@@ -57,8 +57,15 @@ export function QuestionPage(props: QuestionPageProps) {
 
   useEffect(() => {
     // todo replace with values fetched from database
+    fetch(`http://localhost:8000?slug=${slug}`)
+      .then((res) => res.json())
+      .then((data) => {
+        const left = data.find((v) => v.position === 0) || { count: 0 }
+        const right = data.find((v) => v.position === 1) || { count: 0 }
+        setVoteValues([+left.count, +right.count])
```

Comment:
Maybe using Number(left.count) is a better way. It's faster than a parseInt and it avoid radix issue with the + operator